### DISCUSSION TITLE
Make fuzzy match candidates appear in pop-up

### DISCUSSION THREAD

**Discussion by LiGuoV** [company-mode/company-mode#1303] [Q&A]
When I press  "tab"  the "fuzzy match buffer" appears
I want   pop-up fuzzy match candidates ...
I don't know...
thx
![image](https://user-images.githubusercontent.com/35767550/151053720-6d6ca4fc-7606-4147-b409-09a7d972faf2.png)

**Comment by dgutov** [comment]
Have you read the intro at https://company-mode.github.io/ ?
